recipe/meta.yaml

```
@@ -1,34 +1,35 @@
-{% set version = "0.1.40" %}
-{% set sha256 = "385dfb965eca75164d256486c7cf9bae772d24144249fd18b9d15d3cffb70eea" %}
+{% set name = "python-debian" %}
+{% set version = "1.0.1" %}
 
 package:
-  name: python-debian
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/python-debian/python-debian-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3ada9b83a3d671b58081782c0969cffa0102f6ce433fbbc7cf21275b8b5cc771
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
     - python
     - wheel
-    - setuptools
+    - setuptools >=61
+    - setuptools_scm >=6.2
```

https://salsa.debian.org/python-debian-team/python-debian/-/blob/1.0.1/pyproject.toml?ref_type=tags#L5
```suggestion
    - setuptools_scm >=6.2
    - tomli
```